lib/benchee/conversion/scale.ex

```diff
@@ -117,8 +162,8 @@ defmodule Benchee.Conversion.Scale do
   end
 
   # Fetches the magnitude for the given unit
-  defp magnitude(module, unit) do
-    Unit.magnitude(module, unit)
+  defp magnitude(_module, unit) do
```

Comment:
`defp magnitude(_module, %Unit{magnitude: magnitude}) do`

And remove the unused module argument :)